docs/shard.yml.adoc

```diff
@@ -0,0 +1,392 @@
+= shard.yml(5)
+:date: 2020-11-13
+:shards_version: 0.11.0
```

Comment:
is this information right or should it get from the `SHARDS_VERSION` set in the Makefile?

Reply:
Oh yeah, this can be automated. IIRC I didn't implement that right away because I wasn't sure it's a good idea because it could propagate incorrect information if the documentation is not accurate for the newest version. But that's probably hard to coordinate anyways, so it might be better to assume we keep the docs always up to date and it always displays the most recent version.